platforms/gke/base/use-cases/federated-learning/examples/nvflare-tff/README.md

```diff
@@ -0,0 +1,281 @@
+   ```
+
+1. Load NVIDIA FLARE example environment configuration:
+
+   ```bash
+   source "${ACP_PLATFORM_BASE_DIR}/use-cases/federated-learning/examples/nvflare-tff/setup-environment.sh" && load_fl_terraform_outputs
+   ```
+
+1. Run an NVIDIA FLARE container based on the container image you built in the
+   preceding section:
+
+   ```bash
+   docker run --rm -v "${HOME}/nvflare-workspace:/opt/NVFlare/workspace" -it "${NVFLARE_EXAMPLE_CONTAINER_IMAGE_LOCALIZED_ID_WITH_TAG}" bash
+   ```
+
+1. Run the NVIDIA FLARE provisioning tool to create a NVIDIA FLARE project
+   configuration file (`project.yml`) that you can customize to your needs:
+
+   ```bash
+   nvflare provision
+   ```
+
+   When prompted, choose the non highly-available deployment option because
+   highly-available NVIDIA FLARE deployments are not yet supported on
+   Kubernetes.
+
+1. Run the provisioning tool again to generate deployment descriptors:
+
+   ```bash
+   nvflare provision
+   ```
+
+   The provisioning tool generates deployment descriptors:
+
+   - `server1` is the server that will aggregate all the results from the
+     computation
+   - `site-1` and `site-2` are the clients that will be connected to the server
+   - `admin@nvidia.com` is the administration client to start and list jobs
+
+1. Exit the NVIDIA FLARE container:
+
+   ```bash
+   exit
+   ```
+
+1. Copy the workspace folder to Cloud Storage:
+
+   ```bash
+   gcloud storage cp --recursive "${HOME}/nvflare-workspace/workspace" "gs://${NVFLARE_EXAMPLE_WORKSPACE_BUCKET_NAME}"
+   ```
+
+### Push the example container image
+
+In this section, you run a script that pushes the container images to the
+Artifact Registry repository that the Federated learning reference architecture
+provides.
+
+To run the script, do the following:
+
+1. Open [Cloud Shell](https://cloud.google.com/shell).
+
+1. Run the script to build the container image:
+
+   ```sh
+   "${ACP_PLATFORM_BASE_DIR}/use-cases/federated-learning/examples/nvflare-tff/push-container-image.sh"
+   ```
+
+### Verify that NVIDIA FLARE pods are running
+
+In this section, you verify that you deployed NVIDIA FLARE in the reference
+architecture:
+
+1. Open the
+   [GKE Workloads Dashboard](https://cloud.google.com/kubernetes-engine/docs/concepts/dashboards#workloads)
+   and verify that NVIDIA FLARE pods are running in the `fl-1` namespace. The
+   output to look for is similar to the following:
+
+   ```bash
+   Name                               Status Pods
+   nvflare-client1-57d5b45d84-bmv58   OK     1/1
+   nvflare-client2-895b65d8f-p4fs9    OK     1/1
+   nvflare-server1-66c44ddb47-dhtqz   OK     1/1
+   ```
+
+### Submit the training job
+
+In this section, you submit and run a training job:
+
+1. Open [Cloud Shell](https://cloud.google.com/shell).
+
+1. Configure `kubectl` to access the cluster:
+
+   ```bash
+   gcloud container clusters get-credentials <CLUSTER_NAME> --region <CLUSTER_REGION> --project <CLUSTER_PROJECT> --dns-endpoint
```

Comment:
You could populate these as environment variables with the `set_environment_variables.sh` helper script.